### PR TITLE
New version: VSCodium.VSCodium version 1.79.2.23166

### PR DESCRIPTION
**Pull request by daiyam**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109909&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/v/VSCodium/VSCodium/1.79.2.23166/VSCodium.VSCodium.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /mergetasks=!runcode
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/v/VSCodium/VSCodium/1.79.2.23166/VSCodium.VSCodium.installer.yaml:15]:
```suggestion
- Architecture: arm64
```

File: manifests/v/VSCodium/VSCodium/1.79.2.23166/VSCodium.VSCodium.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  InstallerUrl: https://github.com/VSCodium/vscodium/releases/download/1.79.2.23166/VSCodiumSetup-arm64-1.79.2.23166.exe
+  InstallerSha256: 38B55972D02CB6C73536B7251DD359275033188B530B28EDCD790317B95B91C5
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/VSCodium/vscodium/releases/download/1.79.2.23166/VSCodiumSetup-arm64-1.79.2.23166.exe
+  InstallerSha256: 38B55972D02CB6C73536B7251DD359275033188B530B28EDCD790317B95B91C5
+- Architecture: arm64
+  Scope: user
+  InstallerUrl: https://github.com/VSCodium/vscodium/releases/download/1.79.2.23166/VSCodiumSetup-arm64-1.79.2.23166.exe
+  InstallerSha256: 38B55972D02CB6C73536B7251DD359275033188B530B28EDCD790317B95B91C5
```
Comment (stephengillie) [inline manifests/v/VSCodium/VSCodium/1.79.2.23166/VSCodium.VSCodium.installer.yaml:34]:
```suggestion
```
These all appear to be the same `arm64` package.

File: manifests/v/VSCodium/VSCodium/1.79.2.23166/VSCodium.VSCodium.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/v/VSCodium/VSCodium/1.79.2.23166/VSCodium.VSCodium.installer.yaml:15]: Based on the installer name and different hash, I'm guessing this is meant for `User` scope.

**Comment by stephengillie** [comment]
As this package appears to only have `arm64` packages, we might not be able to validate it currently.

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**